src/modules/InstURL.rb

```diff
@@ -157,6 +157,9 @@ def installInf2Url(extra_dir = "")
       @installInf2Url = add_extra_dir_to_url(@installInf2Url, extra_dir) unless extra_dir.empty?
       @installInf2Url = add_ssl_verify_no_to_url(@installInf2Url) unless SSLVerificationEnabled()
 
+      # escape spaces otherwise it would be invalid and rejected by libzypp
+      @installInf2Url.gsub!(" ", "%20")
```

Comment:
is it already stripped? or can it contain space at start and end? if so, then I think we should strip it at first and then do escaping.

Reply:
It should not contain any beginning or trailing space, but you are right, it would be better to remove it.